index.coffee

```diff
@@ -0,0 +1,65 @@
+through = require 'through'
+request = require 'request'
+
+module.exports = ({ url, maxRetries = 10 }) ->
```

Comment:
It'd be nice if this was an `opt` object so that we can potentially pass through more options to the underlying request module like headers and such